payjoin/src/psbt/mod.rs

```diff
@@ -254,8 +254,8 @@ impl fmt::Display for InternalPsbtInputError {
             Self::UnequalTxid => write!(f, "transaction ID of previous transaction doesn't match one specified in input spending it"),
             Self::SegWitTxOutMismatch => write!(f, "transaction output provided in SegWit UTXO field doesn't match the one in non-SegWit UTXO field"),
             Self::AddressType(_) => write!(f, "invalid address type"),
-            Self::NoRedeemScript => write!(f, "provided p2sh PSBT input is missing a redeem_script"),
-            Self::InvalidScriptPubKey(e) => write!(f, "provided script was not a valid type of {e}")
+            Self::InvalidScriptPubKey(e) => write!(f, "provided script was not a valid type of {e}"),
+            Self::WeightError(e) => write!(f, "{e}"),
```

Comment:
Can you provide a description of the error here like the ones above.

Reply:
I think the description from the internal types is sufficient. Should have noticed that before. My bad.